docs/index.md

```diff
@@ -1,91 +1,77 @@
-- `tools.py`: The `web_search` and `parse_content` tools. These enable the agent to retrieve search results and parse them.
-- `agent.py`: The main `WebSearchAgent` implementation of a streaming agent with access to the web.
-- `main.py`: A simple script that runs the agent.
+Example of streaming responses:
 
-### Experimenting with the Agent
+```python
+@openai.call("gpt-4o-mini", stream=True)
+@prompt_template("Write a short story about {topic}")
+def stream_story(topic: str): ...
 
-Try asking the agent various questions. It can:
+for chunk, _ in stream_story("a magical forest"):
+    print(chunk.content, end="", flush=True)
+```
 
-- Answer general knowledge questions
-- Provide summaries of current events
-- Explain complex topics
-- And much more!
+## Choose Your Path
 
-The agent will use its web search capability when it needs additional information.
+### For Newcomers
+1. Explore our Jupyter notebooks:
+   - [Quickstart Guide](https://github.com/Mirascope/mirascope/blob/dev/examples/getting_started/quickstart.ipynb): Comprehensive overview of core features
+   - [Dynamic Configuration and Chaining](https://github.com/Mirascope/mirascope/blob/dev/examples/getting_started/dynamic_configuration_and_chaining.ipynb): Various examples ranging from basic usage to more complex chaining techniques
+   - [Tools and Agents](https://github.com/Mirascope/mirascope/blob/dev/examples/getting_started/tools_and_agents.ipynb): Learn to build advanced AI agents
+   - [Evaluation Techniques](https://github.com/Mirascope/mirascope/blob/dev/examples/getting_started/evaluation.ipynb): Assess and improve LLM outputs
+2. Explore our [Cookbook](./cookbook/index.md): Advanced patterns and real-world applications
 
-### Key Mirascope Features Demonstrated
+### For Experienced Developers
+1. Check out our [Learn Documentation](./learn/index.md): In-depth exploration of Mirascope capabilities
+2. Refer to the [API Reference](./api/index.md): Detailed information on classes and functions
+3. Explore our [Cookbook](./cookbook/index.md): Advanced patterns and real-world applications
 
-This tutorial showcases several key Mirascope features:
+## Key Features
 
-1. **LLM Calls**: The `call` decorator simplifies interactions with LLM APIs.
-2. **Prompt Templates**: Dynamic prompt creation using the `@prompt_template` decorator.
-3. **Custom Tools**: Implementation of web search and content parsing tools.
-4. **Streaming**: Real-time responses using Mirascope's streaming capabilities.
-5. **Agent State Management**: Maintaining conversation history and context, including iteratively calling and inserting tools.
+- LLM Calls: Simplified API interactions across multiple providers
+- Prompt Templates: Dynamic prompt creation with type-safe templating
+- Custom Tools: Extend LLM capabilities with your own functions
+- Streaming: Real-time responses for improved user experience
+- Agent State Management: Maintain conversation context and history effortlessly
```

Comment:
nit: i would maybe make the feature "Agent: Effortless manage state and integrate custom tools" instead? what do you think?